src/TransitorySensitivities.jl

```diff
@@ -205,6 +208,11 @@ function normalizeadjointtransitorysensitivities!(dSdt,ssys::SystemSimulation,t,
             dSdt ./= y[ind]
         end
     end
+    for (i,inter) in enumerate(ssys.interfaces)
+        if isa(inter,AbstractReactiveInternalInterface)
```

Comment:
Can you add blanks after the commas here

Reply:
added